### PR TITLE
Fs 3281 refactor flag endpoints

### PR DESCRIPTION
**Pull request by Adam-W1**
https://dluhcdigital.atlassian.net/browse/FS-3281

### Change description
- Old flagging endpoint is obsolete now. Once we archived the single old flag application histories(FS-3008).  We need to remove old flagging endpoints, functions & update tests.

- Remove old flag endpoints, functions and tests.

- Migration script to drop old flag tables 

Make sure unit tests are updated to flags_v2

- [x] Unit tests and other appropriate tests added or updated
- [x] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
pytest

### Screenshots of UI changes (if applicable)
image

### CODE REVIEW

File: config/envs/default.py [hunk 1]
```diff
@@ -98,15 +98,15 @@ class DefaultConfig:
         "/assessments/get-stats/{fund_id}/{round_id}?{params}"
     )
 
-    ASSESSMENTS_STATS_FLAGS_V2_ENDPOINT = (
-        "/assessments/get-stats_flags_v2/{fund_id}/{round_id}?{params}"
+    ASSESSMENTS_STATS_FLAGS_ENDPOINT = (
```
Comment (RamuniN) [inline config/envs/default.py:101]: looks like you missed refactoring this....it's duplicated on line 98

File: config/envs/default.py [hunk 2]
```diff
@@ -98,15 +98,15 @@ class DefaultConfig:
...
+        "/assessments/get-stats/{fund_id}/{round_id}?{params}"
     )
 
     APPLICATION_OVERVIEW_ENDPOINT_FUND_ROUND_PARAMS = (
         "/application_overviews/{fund_id}/{round_id}?{params}"
     )
-    APPLICATION_OVERVIEW_FLAGS_V2_ENDPOINT_FUND_ROUND_PARAMS = (
-        "/application_overviews_flags_v2/{fund_id}/{round_id}?{params}"
+    APPLICATION_OVERVIEW_FLAGS_ENDPOINT_FUND_ROUND_PARAMS = (
+        "/application_overviews/{fund_id}/{round_id}?{params}"
```
Comment (RamuniN) [inline config/envs/default.py:109]: duplicated on line 106

File: app/assess/helpers.py [hunk 3]
```diff
@@ -129,26 +129,26 @@ def is_flaggable(flag_status: str):
...
-            (FlagV2.from_dict(flag) if isinstance(flag, dict) else flag)
+            (Flag.from_dict(flag) if isinstance(flag, dict) else flag)
             for flag in Flags
         ]
         if Flags
         else []
     )
     all_latest_status = [flag.latest_status for flag in flags_list]
-    return FlagTypeV2.QA_COMPLETED in all_latest_status or False
+    return FlagType.QA_COMPLETED in all_latest_status or False
```
Comment (RamuniN) [inline app/assess/helpers.py:143]: we can remove this function. `QA_COMPLETED` status is not a part of flags status anymore

**Review by RamuniN — COMMENTED** [review]
Few suggestions, Rest of them looks good to me.

File: config/envs/default.py (hunk 1 above)
Comment (Adam-W1) [reply config/envs/default.py:101]: good spot

Comment (RamuniN) [reply config/envs/default.py:97]: can we name it `APPLICATION_OVERVIEW_ENDPOINT_FUND_ROUND_PARAMS` like before ? this is not a flag specific endpoint right

File: config/envs/default.py [hunk 4]
```diff
@@ -94,20 +94,13 @@ class DefaultConfig:
     APPLICATION_SEARCH_ENDPOINT = CommonConfig.APPLICATION_SEARCH_ENDPOINT
 
     # Assessment store endpoints
-    ASSESSMENTS_STATS_ENDPOINT = (
+    ASSESSMENTS_STATS_FLAGS_ENDPOINT = (
```
Comment (RamuniN) [inline config/envs/default.py:97]: can we name it to `APPLICATION_OVERVIEW_ENDPOINT_FUND_ROUND_PARAMS` like before ? this is not a flag specific endpoint right

Comment (Adam-W1) [reply config/envs/default.py:97]: of course, good shout

**Review by RamuniN — DISMISSED** [review]
LGTM👍

**Review by RamuniN — APPROVED** [review]
👍

**Pull request merged**